hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/HRegion.java

```diff
@@ -6201,7 +6205,10 @@ String prepareBulkLoad(byte[] family, String srcPath, boolean copyFile)
     boolean isSuccessful = false;
     try {
       this.writeRequestsCount.increment();
-
+      if (rsServices != null && rsServices.getMetrics() != null) {
+        rsServices.getMetrics().updateWriteQueryMeter(this.htableDescriptor.
+          getTableName());
+      }
```

Comment:
The write meter update should place after write execution, it is different from `writeRequestsCount` because execution time matters.